CONTRIBUTING.md

```diff
@@ -6,6 +6,13 @@ this case, the [Apache Software License v2](LICENSE).
 
 ## Preparing your development environment
 
+Sygnal depends on the `psycopg2` database adapter for PostgreSQL. 
+You may need to install development headers for Python and libpq. For example on Debian/Ubuntu these can be installed with:
```

Comment:
please could you wrap this line at 80 characters?

Reply:
sorry, looked like I did last time. Let me check again and push.